tests/x11/evolution/evolution_prepare_servers.pm

```diff
@@ -102,7 +102,7 @@ sub run() {
 }
 
 sub test_flags() {
-    return is_public_cloud() ? {milestone => 0, fatal => 1, no_rollback => 1} : {milestone => 1, fatal => 1};
+    return is_public_cloud() ? {milestone => 0, fatal => 1, no_rollback => 1} : {milestone => 1, fatal => 0};
```

Comment:
this test is not running for public cloud , can you please also just drop condition ? 
```suggestion
    return {milestone => 1, fatal => 0};
```

Reply:
where you see request to add the test ?

Reply:
Done